src/vorta/models.py

```diff
@@ -241,6 +241,8 @@ def init_db(con):
     # Create missing settings and update labels. Leave setting values untouched.
     for setting in get_misc_settings():
         s, created = SettingsModel.get_or_create(key=setting['key'], defaults=setting)
+        if created and setting['key'] == "use_light_icon":
```

Comment:
Instead of adding a condition, why not change the default setting directly [here](https://github.com/borgbase/vorta/blob/929248deee35d2ebd9a1dad1a07426fe749f44c3/src/vorta/models.py#L193)? E.g.

```
        {
            'key': 'use_light_icon',
            'value': detect_use_light_icon(),
...
```

Reply:
Because I put it in a stupid place in `src/vorta/views/misc_tab.py`.

Performance-wise you're right. Then keep it as you first had it.